Makefile

```diff
@@ -655,6 +668,7 @@ bare: clean
 	$(verbose) rm -f $(mod_conf)
 	$(verbose) rm -f $(booleans)
 	$(verbose) rm -fR $(htmldir)
+	$(verbose) rm -fR $(udicatemplatesdir)
```

Comment:
I don't think this should be added because `make bare` doesn't delete installed files.

Reply:
Good point. I've removed this.